src/assets/stylesheets/components/_lazy-loader.scss

```diff
@@ -0,0 +1,103 @@
+.container-lazy-loader {
+  height: 100vh;
+  padding: rem(32px);
+
+  background: map-get($brand-black, 'light-1');
+}
+
+.lazy-loader-header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+
+  &__web-icon .react-loading-skeleton {
+    width: rem(117px);
+    height: rem(18px);
+
+    border-radius: rem(14px);
+  }
+
+  &__user-image .react-loading-skeleton {
+    width: rem(36px);
+    height: rem(36px);
+
+    border-radius: 50%;
+  }
+}
+
+.lazy-loader-body {
+  display: flex;
+  flex-direction: column;
+  max-width: rem(768px);
+  margin-top: rem(49px);
+  margin-right: auto;
+  margin-left: auto;
+
+  &__date .react-loading-skeleton {
+    width: rem(117px);
+    height: rem(18px);
+
+    border-radius: rem(14px);
+  }
+
+  &__day .react-loading-skeleton {
+    width: rem(90px);
+    height: rem(18px);
+    margin-top: rem(14px);
+
+    border-radius: rem(14px);
+  }
+
+  &__survey-image .react-loading-skeleton {
+    height: rem(200px);
+    margin-top: rem(64px);
+
+    border-radius: rem(14px);
+
+    @include media-breakpoint-up(sm) {
+      height: rem(305px);
+    }
+  }
+}
+
+.lazy-loader-footer {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-top: rem(38px);
```

Comment:
This margin should be `32px`
![image](https://user-images.githubusercontent.com/14927672/123556005-d69da580-d7aa-11eb-966e-d78600ffab2a.png)

Reply:
![Screen Shot 2564-06-28 at 16 17 40](https://user-images.githubusercontent.com/29707647/123612019-5fa8f100-d82c-11eb-9420-a8ae138ca357.png)
I think it should be 38px according to the image

Reply:
I agree. It should be `38px`. But with the current margin and also there is some line height is making the distance much larger. Below two images are the current distance differences -
- Difference between `&__survey-title` and `&__survey-image`
![image](https://user-images.githubusercontent.com/14927672/123695193-141e3380-d87c-11eb-99ae-8f6b44b3978a.png)
- Difference between `&__button` and `&__survey-image`
![image](https://user-images.githubusercontent.com/14927672/123695249-24cea980-d87c-11eb-80c9-2cdf15bbdd95.png)

You can use chrome extension - [MeasureMate](https://chrome.google.com/webstore/detail/measuremate/fcbffocoaopnenhgbchlchgndohggdei) to easily measure these distances. It's really helpful. 

I also think difference between `&__survey-title` and `&__survey-subtitle` is a bit larger. I cannot say for certain because the differences are not shown in figma. 

The below style changes should fix the issue. You will get nice `38px` distance as well as the `32px`.

```
&__footer {
  margin-top: rem(32px);
  line-height: 0;
}

&__survey-subtitle .react-loading-skeleton {
  margin-top: rem(8px);
}
```

Reply:
Thanks for the suggestion. Fixed in de50d2fca76f0e53cbd12089380862b11fb6d020